src/modules/launcher/Wox.Plugin/ThemeManager.cs

```diff
@@ -90,25 +93,17 @@ private static Theme GetHighContrastBaseType()
                 case "hcblack":
                     return Theme.HighContrastBlack;
                 default:
-                    return Theme.None;
+                    return Theme.HighContrastOne;
             }
         }
 
         private void ResetTheme()
         {
-            if (WindowsThemeHelper.IsHighContrastEnabled())
-            {
-                Theme highContrastBaseType = GetHighContrastBaseType();
-                ChangeTheme(highContrastBaseType);
-            }
-            else
-            {
-                string baseColor = WindowsThemeHelper.GetWindowsBaseColor();
-                ChangeTheme((Theme)Enum.Parse(typeof(Theme), baseColor));
-            }
+            var settings = _settingsUtils.GetSettings<PowerLauncherSettings>(PowerLauncherSettings.ModuleName);
```

Comment:
In our current architecture we do not read the settings directly by using settings classes. Instead we use `SettingsWatcher` for this. The usual flow of changes is something like : 
1.  Update the settings.json file from Settings process.
2. These changes will invoke settings watcher.
3. Look for changes in settings watcher and update settings.

My suggestion would be to remove `SettingsUtils` from ThemeManager. Then look for changes in settingswatcher and call `ChangeTheme` function with new value.